Client/Frontend/Strings.swift

```diff
@@ -56,7 +56,28 @@ private func MZLocalizedString(
 // MARK: - Alerts
 extension String {
     public struct Alerts {
-
+        public struct RestoreTabs {
+            public static let Title = MZLocalizedString(
+                "Alerts.RestoreTabs.Title.v109",
+                tableName: "Alerts",
+                value: "Firefox crashed. Restore your tabs?",
+                comment: "The title of the restore tabs pop-up alert.")
+            public static let Message = MZLocalizedString(
+                "Alerts.RestoreTabs.Message.v109",
+                tableName: "Alerts",
+                value: "Sorry about that. Restore tabs to pick up where you left off.",
+                comment: "The body of the restore tabs pop-up alert.")
```

Comment:
I think we could add a little more context in this comment, so something like:
"The body of the restore tabs pop-up alert. This alert shows when Firefox crashes" (or something along those lines if what I put there is not accurate)